interfaces/apparmor/spec_test.go

```diff
@@ -506,6 +506,33 @@ func (s *specSuite) TestApparmorOvernameSnippets(c *C) {
 	c.Assert(updateNS[0], Equals, profile)
 }
 
+func (s *specSuite) TestApparmorExtraLayouts(c *C) {
+	snapInfo := snaptest.MockInfo(c, snapTrivial, &snap.SideInfo{Revision: snap.R(42)})
+	snapInfo.InstanceKey = "instance"
+
+	restore := apparmor.SetSpecScope(s.spec, []string{"snap.some-snap_instace.app"})
+	defer restore()
+
+	extraLayouts := []snap.Layout{
+		{
+			Path: "/test",
+			Bind: "/usr/home/test",
+			Mode: 0755,
+		},
+	}
+
+	s.spec.AddExtraLayouts(snapInfo, extraLayouts)
+	c.Assert(s.spec.Snippets(), HasLen, 0)
+
+	updateNS := s.spec.UpdateNS()
+	c.Assert(updateNS, HasLen, 10)
```

Comment:
(nitpick) why 10 here? does it just happen to be 7 without the extra layout?

Reply:
Added a comemnt explaining this, it adds 10 lines because of reverse traversal of the path /usr/home/test - so the first 3 lines are the mount stuff, and then the rest is permissions for the prefix